scss/_buttons.scss

```diff
@@ -79,19 +82,23 @@
   &:hover {
     color: $btn-link-hover-color;
     text-decoration: $link-hover-decoration;
+    border-color: transparent;
   }
 
   &:focus,
   &.focus {
     text-decoration: $link-hover-decoration;
+    border-color: transparent;
   }
 
   &:disabled,
   &.disabled {
     color: $btn-link-disabled-color;
   }
 
-  // No need for an active state here
+  &:active {
+    border-color: transparent;
+  }
```

Comment:
`border`s on button seems unrelated, aren't they?

Reply:
Yeah, I know, was just trying things out to decrease the number of unique colors.